src/System.Collections.Specialized/src/System.Collections.Specialized.builds

```diff
@@ -4,8 +4,14 @@
   <ItemGroup>
     <Project Include="System.Collections.Specialized.csproj" />
     <Project Include="System.Collections.Specialized.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
+    </Project>
+    <Project Include="System.Collections.Specialized.csproj">
       <TargetGroup>net46</TargetGroup>
```

Comment:
Correct you will need to keep the net463 one also. @weshaggard looks like System.Collections.Specialized is a partial facade, so given that it does contain implementation wouldn't it be worth keep producing the netstandard1.3 version as part of the build? I talked with @ericstj and the basic guideline was that anything that is a full facade is fine to remove shipped versions, but every library that has full/partial implementation on it, it might be worth keeping the x-compilation for bug-fixing.